search-commons/src/main/java/no/unit/nva/search/SearchClient.java

```diff
@@ -142,8 +145,14 @@ private SearchRequest getSearchRequest(String term, int results, int from, Strin
 
 
     private SearchResourcesResponse toSearchResourcesResponse(String searchterm, String body)
-            throws JsonProcessingException {
-        JsonNode values = objectMapperWithEmpty.readTree(body);
+            throws SearchException {
+
+        JsonNode values;
+        try {
+            values = objectMapperWithEmpty.readTree(body);
+        } catch (JsonProcessingException e) {
+            throw new SearchException("Unable to parse SearchResponse", e);
+        }
```

Comment:
```suggestion
        JsonNode values = attempt(() -> objectMapperWithEmpty.readTree(body)).orElseThrow();
```
Magic string indicates that no test is written for this exception. In addition since SearchResponse is throwing a 500 error, there is no reason to throw that exception. Also, I do not think that this exception will ever be thrown.